src/vm/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "vmadm",
   "description": "administrative tool(s) for managing VMs",
   "version": "1.0.5",
```

Comment:
Version bump?

Reply:
This looks like the same situation as the `fw` module. I don't believe it's used outside of smartos live, a version bump wouldn't hurt but I don't think it's necessary either.